requirements/edx/base.txt

```diff
@@ -56,7 +56,7 @@ nose==1.3.3
 oauthlib==0.6.3
 paramiko==1.9.0
 path.py==3.0.1
-Pillow==2.6.1
+Pillow==2.7.0
```

Comment:
This comes from the differences on the birch branch vs master. It should be gone when I rebase this to master.